demoapp/demo_app_functions.py

```diff
@@ -450,4 +466,479 @@ def update_csv(bill_num: str, title: str, summarized_bill: str, category: str, t
     
     df.to_csv(csv_file_path, index=False)
     return df
+
+class Container: 
+
+    '''
+    A container class to store all the details pertinant to that specific action
+    '''
+
+    def __init__(self, **kwargs):
+        for k, v in kwargs.items(): setattr(self, k, v)
+
+class BillDetails(Container): pass
```

Comment:
We need to be explicit about the data structure and its intended input signature.